src/uct/api/uct.h

```diff
@@ -210,6 +213,20 @@ typedef enum {
 } uct_alloc_method_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief  Wakeup event types.
+ */
+enum uct_wakeup_event_types {
+    UCT_WAKEUP_TX_COMPLETION   = UCS_BIT(0),
+    UCT_WAKEUP_TX_RESOURCES    = UCS_BIT(1),
+    UCT_WAKEUP_RX_AM           = UCS_BIT(2),
+    UCT_WAKEUP_RX_SIGNALED_AM  = UCS_BIT(3),
+    UCT_WAKEUP_TX_ERROR        = UCS_BIT(4),
```

Comment:
pls remove the event types which we don't implement/use in this PR and add them later on.